pdns/dnsdist.hh

```diff
@@ -298,7 +298,10 @@ struct DNSDistStats
     {"fd-usage", getOpenFileDescriptors},
     {"dyn-blocked", &dynBlocked},
     {"dyn-block-nmg-size", [](const std::string&) { return g_dynblockNMG.getLocal()->size(); }},
-    {"security-status", &securityStatus}
+    {"security-status", &securityStatus},
+    // Latency histogram
+    {"latency-sum", &latencySum},
+    {"latency-count", &responses}, // This is the same metric as responses but uses the metric name as expected by Prometheus
```

Comment:
I'm afraid this is not accurate, we do update the latency metrics for cache-hits and self-generated answers (see `prepareOutgoingResponse()` and the `responses` counter is not incremented in these cases.

Reply:
`g_stats.responses` + `g_stats.selfAnswered` + `g_stats.cacheHits` would be closer to the value you are looking for.

Reply:
Oh okay didn't know that, will try and change that

Reply:
Now added a stats function which returns the sum of those 3 values